common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java

```diff
@@ -77,7 +78,10 @@ public void render(@NotNull GuiGraphics graphics, int i, int j, float f) {
     }
 
     protected void renderBg(GuiGraphics graphics, float partialTick, int mouseX, int mouseY) {
-        graphics.fill(0, 0, width, height, 0xD0000000);
+        RenderSystem.enableBlend();
+        RenderSystem.defaultBlendFunc();
+        graphics.blitRepeating(HEADING, 0, 0, width, height,0, 128, 128, 128);
+        RenderSystem.disableBlend();
         graphics.blitRepeating(HEADING, 0, 0, this.width, 15, 0, 0, 128, 15);
         if (drawSidebar()) {
             int sidebarWidth = (int) (this.width * 0.25f) - 2;
```

Comment:
I reckon it's worth moving textures to give you some more freedom, and then use that freedom to have a seperately drawn sidebar texture.

So just under HEADING, you can add

`public static final ResourceLocation BACKGROUND = new ResourceLocation(Heracles.MOD_ID, "textures/gui/background.png");`

then add that png as a 256x256 and make the top half the transparent black color

and then here, you can just:

```suggestion
        RenderSystem.enableBlend();
        RenderSystem.defaultBlendFunc();
        int sidebarWidth = (int) (this.width * 0.25f) - 2;
        if (drawSidebar()) {
            graphics.blitRepeating(BACKGROUND, 0, 0, sidebarWidth, height, 0, 0, 128, 128);
            graphics.blitRepeating(BACKGROUND, sidebarWidth, 0, width - sidebarWidth, height, 128, 0, 128, 128);
        } else {
            graphics.blitRepeating(BACKGROUND, 0, 0, width, height, 128, 0, 128, 128);
        }
        RenderSystem.disableBlend();
        graphics.blitRepeating(HEADING, 0, 0, this.width, 15, 0, 0, 128, 15);
        if (drawSidebar()) {
```

Reply:
thinking about this a bit harder, moving this from header might be wrong.